error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/PreconditionsRules.java

```diff
@@ -52,6 +55,26 @@ void after(boolean condition, String message) {
     }
   }
 
+  /**
+   * Prefer {@link Preconditions#checkArgument(boolean, String, Object...)} over more verbose
+   * alternatives.
+   */
+  // XXX: This check assumes that the format string only uses `%s` placeholders.
```

Comment:
So this isn't fully correct. One option is to create an BugChecker instead, but that's more work :thinking:

Reply:
Okay, I went down the rabbit hole: #1139.